src/styles/blocks/header.scss

```diff
@@ -0,0 +1,65 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px 0 #0000000d;
+
+  &__logo {
+    display: flex;
+    margin-left: 50px;
+  }
+
+  &__nav {
+    margin-right: 50px;
+  }
+
+  &__list {
+    display: flex;
+    justify-content: space-between;
+    list-style: none;
+  }
+
+  &__item {
+    position: relative;
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+    height: 60px;
+    margin-left: 20px;
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    color: #060b35;
+    text-decoration: none;
+    text-transform: uppercase;
+    transition: $transition-smooth;
+
+    &:hover {
+      color: #00acdc;
+    }
+  }
+}
+
+.is-active {
+  display: flex;
+  align-items: center;
```

Comment:
```suggestion
  align-items: center;
  justify-content: center;
```

Reply:
When changing the value of justify-content: center; in the header, everything breaks in the header 
![image](https://github.com/mate-academy/layout_catalog/assets/138622358/fac40550-b06a-4982-870f-540741608b1f)

Reply:
The `is-active` class should highlight the active links. For other links, it doesn't work properly. Consider fixing it
<img width="524" alt="Screenshot 2024-06-16 at 17 22 58" src="https://github.com/mate-academy/layout_catalog/assets/135232001/85858ea3-85fb-4c3e-a0d4-4fda0925644f">

Reply:
I don't understand how to do this using the is-active class in SCSS without using JavaScript. Should I write the hover styles separately for each link with such styling instead?